extensions/php/languages/phpdoc/highlights.scm

```diff
@@ -0,0 +1,13 @@
+(tag_name) @keyword
```

Comment:
This was copied from the jsdoc injection: https://github.com/zed-industries/zed/blob/main/crates/languages/src/jsdoc/highlights.scm#L1